app/src/__mocks__/popper.js.js

```diff
@@ -0,0 +1,15 @@
+// __mocks__/popper.js.js
```

Comment:
can we add the url to the stack overflow post as to why we needed to add this file? :)